src/components/Card.spec.js

```diff
@@ -27,21 +27,17 @@ describe('Card', () => {
     expect(explanation).toBeInTheDocument();
   });
 
-  it('calls onClick when clicking the delete or bookmark icon', () => {
+  it('calls onClick when clicking the bookmark icon', () => {
```

Comment:
```suggestion
  it('calls Callback when clicking the bookmark icon', () => {
```

Reply:
You can also call it onClick, but better make sure to use the same name inside of your component so that everybody knows where to look.